Practical 1 on segmentation added.

annnyway opened the pull request:
Hello, Fran! Here is my report on Practical 1. I'm so sorry to be late. It was very new and very difficult for me, but I will work on it more. I hope you will check my report. I will be waiting for your comments. 

Thank you in advance. 
Ann Nikolaeva

annnyway committed:
quiz 1 added

annnyway committed:
practical-segmentation added

annnyway committed:
Upgrade segmentation-response-NikolaevaAnn.md

slight changes

annnyway committed:
slight changes in practical 1

annnyway committed:
slight changes in practical 1

annnyway committed:
slight changes in Practical 1 again

ftyers commented:
Comment: For pragmatic_segmenter did you use the English rules ? I don't think there are Ukrainian rules, but perhaps it might work better with the Russian rules than the English rules?

annnyway commented:
I followed the given instructions on the practical: https://ftyers.github.io/2018-komp-ling/practicals/segmentation.html and didn't know that I could change the rules, but now I see that maybe I could change the language in segmenter.rb file to Russian. Will it change the English rules to the Russian ones? I will try it today or tomorrow. 

Thank you!

annnyway committed:
Segmentation-response updated

Added the result of Pragmatic Segmenter after changing its language into Russian

annnyway commented:
I have added more results of Pragmatic Segmenter. Do I need to make a new pull-request?

annnyway committed:
transliteration-response added

annnyway committed:
Update transliteration-response.md

annnyway committed:
Update transliterate.py

annnyway committed:
quiz-2 added

annnyway committed:
some fixes

annnyway committed:
some fixes again

annnyway committed:
Update quiz2-response.md

annnyway committed:
Delete fst1.jpg

annnyway committed:
Delete fst2.jpg

annnyway committed:
Delete fst3.jpg

annnyway committed:
add fst diagrams

annnyway committed:
unigram-response added

annnyway committed:
quiz-3 added

annnyway committed:
Update quiz-3.md

annnyway committed:
Update quiz-3.md

annnyway committed:
Update quiz-3.md

annnyway committed:
Update rank.py

annnyway committed:
'some mistakes fixed'

annnyway committed:
practical 4 added

annnyway committed:
Update perceptron.py

annnyway committed:
Update maxmatch.py

Commit:
user name and email changed

Commit:
practical 4 updated

Commit:
practical 4 updated

annnyway committed:
Update unigram_tagger-response.md

Commit:
practical on transliteration updated

annnyway committed:
Update transliteration-response.md

annnyway committed:
Update transliteration-response.md

Commit:
some practicals renamed

annnyway committed:
Delete Punkt.py

annnyway committed:
Delete maxmatch.py

annnyway committed:
Delete segmentation-response.md

Commit:
xrenner response added

annnyway committed:
Update xrenner-response.md

annnyway committed:
Delete azazel.conllu

annnyway committed:
xrenner-response updated
